src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
```

Comment:
Thank you. Looks better. 

How about this: 
If an unanalyzed track is playing, Mixxx won't apply the a newly calculated ReplayGain while it is playing to avoid abrupt volume changes."